components/brave_wallet_ui/components/extension/post-confirmation/confirming/index.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react'
+import { useSelector } from 'react-redux'
+
+// Constants
+import { WalletState } from '../../../../constants/types'
+
+// Hooks
+import { useExplorer } from '../../../../common/hooks'
+
+// Utils
+import { getLocale } from '$web-common/locale'
+
+// Styled components
+import { Panel } from '../..'
+import {
+  ConfirmationsNumber,
+  ConfirmingIcon,
+  ConfirmingText,
+  Title
+} from './style'
+import {
+  ButtonRow,
+  DetailButton,
+  LinkIcon,
+  TransactionStatusDescription
+} from '../common/style'
+
+interface Props {
+  headerTitle: string
+  confirmations: number
+  confirmationsNeeded: number
+  onClose: () => void
+}
+
+const TransactionConfirming = (props: Props) => {
+  const {
+    headerTitle,
+    confirmations,
+    confirmationsNeeded,
+    onClose
+  } = props
+
+  // redux
+  const {
+    selectedNetwork
+  } = useSelector((state: { wallet: WalletState }) => state.wallet)
```

Comment:
directly select `selectedNetwork` instead of all of state.wallet

Reply:
Done.